tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -51,14 +51,17 @@ using tensorflow::tracing::ScopedAnnotation;
 // since we can use timers around thunks.
 GpuExecutable::GpuExecutable(
     const string& text, const std::vector<uint8>& binary,
+    absl::variant<std::pair<int, int>, int> compute_capability,
```

Comment:
I think we may want something like `using GpuVersion = absl::variant<std::pair<int, int>, int>` to make the type more descriptive.

Reply:
I was trying to follow the [naming from upstream ](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/gpu/gpu_executable.h#L146). I'm good with either of those names.

Reply:
you can see something like:

```
  using BufferAllocToDeviceMemoryMap =
      absl::flat_hash_map<BufferAllocation::Index, se::DeviceMemoryBase>;
```

for types with somewhat complicated semantic, and I believe this particular type suffice to be aliased.